tracing-appender/src/lib.rs

```diff
@@ -1,15 +1,80 @@
+//! Writers for logging events/spans
+//!
+//! # Overview
+//!
+//! [`tracing`][tracing] is a framework for instrumenting Rust programs to collect
+//! structured, event-based diagnostic information. This crate provides the ability
+//! for `tracing` events and spans to be recorded in a non blocking manner by using a
+//! dedicated logging thread. It also provides a [`RollingFileAppender`][file_appender]
+//! that can be used with or without the non blocking writer.
+//!
+//! [file_appender]: ./rolling/struct.RollingFileAppender.html
+//! [tracing]: https://docs.rs/tracing/
+//!
+//! # Usage
```

Comment:
Typically, these usage sections begin with 
```suggestion
//! # Usage
//!
//! First, add this to your `Cargo.toml`:
//! ```toml
//! tracing-appender = "0.1"
//! ```
//!
```